docs/alps/vclusters.md

```diff
@@ -29,54 +22,74 @@ Host balfrin* daint* santis* todi*
 
 This would allow standard connections like `ssh santis` but also specifying the login node like `ssh santis-ln002` if needed. Replace `cscsusername` with your actual user name.
 
-## Daint
-
-Daint (Alps) is the vCluster dedicated to the User Lab. It is currently accessible at `daint.alps.cscs.ch` (until the current Piz Daint gets decommissioned), so connect with `ssh daint.alps` with the `ssh` settings above.
+## Santis
 
-Even though Weather and Climate also has the dedicated vCluster Santis (see [below](#santis)), traditional projects might also land on Daint.
+The vCluster `santis` is dedicated to **Climate and Weather**. It might, at the beginning, only host [EXCLAIM :material-open-in-new:](https://c2sm.ethz.ch/research/exclaim.html){:target="_blank"} and related projects.
```

Comment:
```suggestion
The vCluster `Santis` is dedicated to **Climate and Weather** and may initially host only [EXCLAIM :material-open-in-new:](https://c2sm.ethz.ch/research/exclaim.html){:target="_blank"} and related projects.
```

Reply:
I tried to come up with a unified reference to the vClusters, since we call them

- Säntis
- Santis
- `santis`

I would not use `Santis`, however.

Reply:
You are right, it's better to unify. I'm happy with any of the versions.

Reply:
Awesome, I will use `santis` etc. to also keep consistency with CSCS docs.